.aws/src/backfillLambda.ts

```diff
@@ -123,8 +123,8 @@ export class BackfillLambda extends Resource {
       threshold: 4000,
       // rather than the average, we want to track how many messages in total end up in the DLQ
       statistic: 'Sum',
-      alarmActions: [this.pagerDuty.snsNonCriticalAlarmTopic.arn],
-      okActions: [this.pagerDuty.snsNonCriticalAlarmTopic.arn],
+      alarmActions: [this.pagerDuty?.snsNonCriticalAlarmTopic.arn],
+      okActions: [this.pagerDuty?.snsNonCriticalAlarmTopic.arn],
```

Comment:
This change was the last change needed to be able to deploy to Dev - apparently, in the backfill lambda most of this code is copied from alarms were not set up at the time testing on Dev was in progress, they were set up later on when test runs were only done on Prod, so this error has not come up before.